app/adapters/weather/client.rb

```diff
@@ -0,0 +1,33 @@
+module Weather
+  class Client
+    include HTTParty
+    base_uri "https://dataservice.accuweather.com"
+
+    def initialize
+      @apikey = ENV.fetch("API_Key")
```

Comment:
fetch лучше использовать чтобы в переменную записывалось какое-то другое значение, если переменной несуществует, иначе лучше его не юзать

```suggestion
      @apikey = ENV["API_Key"]
```